packages/accessibility/test/behaviors/testDefinitions.ts

```diff
@@ -337,16 +404,18 @@ definitions.push({
       testHelper.convertToMatchingTypeIfApplicable(attributeExpectedValue),
     )
 
-    const propertyFirstPropNegate = {}
-    propertyFirstPropNegate[
-      propertyDependingOnFirst
-    ] = !testHelper.convertToMatchingTypeIfApplicable(attributeExpectedValue)
-    propertyFirstPropNegate[propertyDependingOnSecond] = true
-    const actualResultFirstPropertyNegate = parameters.behavior(propertyFirstPropNegate).attributes
-      .root[attributeToBeAdded]
-    expect(testHelper.convertToMatchingTypeIfApplicable(actualResultFirstPropertyNegate)).toEqual(
-      !testHelper.convertToMatchingTypeIfApplicable(attributeExpectedValue),
-    )
+    if (typeof testHelper.convertToMatchingTypeIfApplicable(attributeExpectedValue) === 'boolean') {
```

Comment:
it worked with other params apart from boolean, but this part does not make sense unless the value is a boolean.